exercises/poker/poker_test.go

```diff
@@ -282,12 +282,10 @@ var invalidTestCases = []struct {
 }
 
 func TestBestHandValid(t *testing.T) {
-	if testVersion != targetTestVersion {
```

Comment:
I admit I was lax here - I should have asked for this to be in a separate commit, in keeping with the idea of having a single logical change per commit. I am sorry for being lax, and accept the responsibility.